tests/testthat/indention_curly_brackets/multi_line_curly_round_spacing-out.R

```diff
@@ -0,0 +1,9 @@
+b <- function(x){ # FIXME indent before {
```

Comment:
Yes. I doubt that that is a very good idea because it is not a general rule to have a space after a round closing parenthesis, e.g. `call1(call2(1, 2), 3)`. We could make it conditional on having `function(` before `)` though. That might work.